sdk/src/wallet/migration/migrate_4.rs

```diff
@@ -69,6 +69,7 @@ impl Migration<crate::client::stronghold::StrongholdAdapter> for Migrate {
     }
 }
 
+#[cfg(feature = "storage")]
 fn migrate_wallet(wallet: &mut serde_json::Value) -> Result<()> {
```

Comment:
Could just move it where it's called since it's only used once but not a big deal